public/css/index.css

```diff
@@ -7,209 +7,152 @@
     --rounded: 999px;
 }
 
-body {
-    background-color: var(--navy-blue) !important;
-}
-
-body div, body img {
+div, img, a, article, label, .post-panel {
     border-radius: 8px;
 }
 
-body a {
-    border-radius: 8px;
+a:hover {
+    text-decoration: underline !important;
 }
 
-body textarea {
-    line-height: 30px;
-    letter-spacing: 2px;
+textarea {
+    line-height: 30px !important;
+    letter-spacing: 2px !important;
+    resize: none;
+    border-radius: 8px;
 }
 
-body p {
+p {
     line-height: 30px;
     letter-spacing: 2px;
     white-space: pre-line
 }
 
-body button {
-    border-radius: 8px;
+button {
+    border-radius: 8px !important;
     color: white;
 }
 
-body button:hover {
+button:hover {
     opacity: 0.8;
 }
 
-body a {
+a {
     text-decoration: none !important;
     color: black;
 }
 
+/* 方形 */
+.square {
+    border-radius: 0;
+}
+
+/* 左下圓角 */
```

Comment:
Try to use english in everywhere we could see in the codebase, including comments.
Meanwhile, I think these `rounded-*` classes could be replaced by tailwind native classes (check [here](https://tailwindcss.com/docs/border-radius))
```html
<Element class="rounded-bl-lg" />
```